Dockerfile

```diff
@@ -2,7 +2,8 @@ FROM node:boron
 
 # Update and install packages
 RUN apt-get update && \
-    apt-get install -y python python-dev python-pip python-virtualenv zip rsync && rm -rf /var/lib/apt/lists/*
+    apt-get install -y python python-dev python-pip python-virtualenv zip rsync default-jdk && \
```

Comment:
@niranjan94 thanks a lot. 
I am not aware of the jdk version. I think Dockerfile installs 1.7 which is not working